### PR TITLE
Pull Request for Issue2046: Adding Zebra + Ge detector to Main

### PR DESCRIPTION
**Pull request by helfrij**
- Created new class BioXASSide32ElementGeDetector, contains all of the Side-specific detector information that was previously in BioXAS32ElementGeDetector. Replaced previous instances of BioXAS32ElementGeDetector with the Side version.
- Created new class BioXASMainInboard32ElementGeDetector.
- Added instance of BioXASZebra, AMZebraDetectorTriggerSource, BioXASMainInboard32ElementGeDetector to BioXASMainBeamline.
- Removed commissioning tool from Main, as it isn't operational with the Zebra yet.

### CODE REVIEW

**Comment by helfrij** [comment]
Hey @iainworkman, would you review?

File: source/beamline/BioXAS/BioXASSide32ElementGeDetector.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#include "BioXASSide32ElementGeDetector.h"
+
+BioXASSide32ElementGeDetector::BioXASSide32ElementGeDetector(const QString &name, const QString &description, AMPVControl *triggerControl, BioXASZebraPulseControl *pulseControl, QObject *parent) :
+	BioXAS32ElementGeDetector(name, description, triggerControl, pulseControl, parent)
+{
+	// Stuff required by AMXRFDetector.
+
+	acquisitionStatusControl_ = new AMReadOnlyPVControl("Status", "DXP1607-I22-01:DetectorState_RBV", this);
+
+	for (int i = 0; i < 20; i++){
```
Comment (iainworkman) [inline source/beamline/BioXAS/BioXASSide32ElementGeDetector.cpp]: How come there are only channels added from 0-19?

**Comment by iainworkman** [comment]
One small question about the number of channels, otherwise it looks good.
Time Estimate: 5 minutes.

File: source/beamline/BioXAS/BioXASSide32ElementGeDetector.cpp (hunk 1 above)
Comment (helfrij) [inline source/beamline/BioXAS/BioXASSide32ElementGeDetector.cpp]: Only the first 20 channels are used on the Side detector. I think this is the way to ensure that the other channels are never used. I'll add a comment, would that make it clearer?

File: source/beamline/BioXAS/BioXASMainBeamline.h [hunk 2]
```diff
@@ -95,6 +96,18 @@ class BioXASMainBeamline : public BioXASBeamline
 	/// Returns the I2 scaler channel detector.
 	virtual CLSBasicScalerChannelDetector* i2Detector() const { return i2Detector_; }
 
+	/// Returns the inboard 32Ge detector.
+	virtual BioXAS32ElementGeDetector* ge32ElementDetector() const { return ge32DetectorInboard_; }
```
Comment (dretrex) [inline source/beamline/BioXAS/BioXASMainBeamline.h]: Wasn't there an issue where we had lists of Ge detectors in BioXASBeamline?  I forget the issue number.  Will this need to be modified to conform to that upon merge?

**Comment by dretrex** [comment]
Looks good.  There is a merge conflict that will need to be addressed.
Time Estimate: 5 minutes

File: source/beamline/BioXAS/BioXASMainBeamline.h (hunk 2 above)
Comment (helfrij) [inline source/beamline/BioXAS/BioXASMainBeamline.h]: You're right, this should be used in the list. Will update.

**Comment by helfrij** [comment]
Merge conflicts should be addressed. Thanks @dretrex @iainworkman!

**Pull request merged**